pkg/cfn/manager/api.go

```diff
@@ -310,6 +311,37 @@ func (c *StackCollection) DescribeStackEvents(i *Stack) ([]*cloudformation.Stack
 	return resp.StackEvents, nil
 }
 
+// LookupCloudTrailEvents looks up stack events in CloudTrail
+func (c *StackCollection) LookupCloudTrailEvents(i *Stack) ([]*cloudtrail.Event, error) {
+	id := i.StackId
+	if id == nil {
+		stack, err := c.describeStack(i)
+		if err != nil {
+			return nil, err
+		}
+		id = stack.StackId
+	}
+
+	input := &cloudtrail.LookupEventsInput{
+		LookupAttributes: []*cloudtrail.LookupAttribute{{
+			AttributeKey:   aws.String(cloudtrail.LookupAttributeKeyResourceName),
+			AttributeValue: id,
+		}},
+	}
+
+	events := []*cloudtrail.Event{}
+
+	pager := func(p *cloudtrail.LookupEventsOutput, last bool) (shouldContinue bool) {
```

Comment:
@christopherhein @tiffanyfay could you please help me understand if this is required, of Go SDK does this automatically for me?

Reply:
From our slack conversation, it turns out we should be using pagination (I'm still not 100% clear, but @tiffanyfay's reading of the docs is same as mine, so better stay on the safe side). I've opened #590 also, so we shall look into this later on for various other API calls.